R/customVarInfo.R

```diff
@@ -312,7 +320,7 @@ defaultVarInfo <- list(
     name = 'Scalar',
     doesApply = function(v) is.atomic(v) && length(v) == 1,
     hasChildren = FALSE,
-    toString = deparse
+    toString = function(v) paste(deparse(v), collapse = '\n')
```

Comment:
Is there any possible case that when the scalar value could produce a character of length > 1?

Reply:
`e <- try({stop('ERROR')})` matches the `doesApply` and returns:
```
> deparse(e)
[1] "structure(\"Error in try({ : ERROR\\n\", class = \"try-error\", condition = structure(list("
[2] "    message = \"ERROR\", call = doTryCatch(return(expr), name, "
[3] "        parentenv, handler)), class = c(\"simpleError\", \"error\", "
[4] "\"condition\")))"
```

Reply:
I suggest that we use `function(v) is.atomic(v) && length(v) == 1 && is.null(attributes(v))` in this case to ensure `scalar` captures only pure scalar value without addiitonal class etc, then a simple `deparse` could work here.